Monika After Story/game/script-topics.rpy

```diff
@@ -546,6 +546,32 @@ label monika_selfesteem:
             m 3hub "I'll always root for you, [player]!"
     return "derandom"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="songwriting", 
+            category=["music"],
+            prompt="Songwriting", 
+            random=True, 
+            pool=True
+        )
+    )
+
+label monika_example:
+    m 3eub "Hey [player], have you ever written a song?"
+    m 3ekb "I've tried myself, but it's hard to do with my limited resources here."
```

Comment:
I'm not really a musician, but I don't think it takes that many resources Monika doesn't have to write a song. 

There's a piano, and sure, there's no paper or anything to record them, but the instrument is there. Not to mention that Monika already wrote a song.

Reply:
You don't really need to write anything down if you practice it often, since it just gets done as muscle memory. So paper may not even be needed.

Reply:
That's true. Back to the drawing, or shall I say, writing board? I'll rework it.